src/Behavioral.Automation/FluentAssertions/Assert.cs

```diff
@@ -112,7 +112,7 @@ private static T TryGetValue<T>(Func<T> getValue, TimeSpan wait, int attempts =
                 {
                     return getValue();
                 }
-                catch (StaleElementReferenceException)
+                catch (Exception e) when (e is StaleElementReferenceException || e is NoSuchElementException)
```

Comment:
We use `Assert.ShouldBecome` to find sub elements, like this `Assert.ShouldGet(() => Element.FindElements(locator))`, but if there is no such element yet, `Assert` fails immediately without retries, which violates `ShouldGet `purpose, IMO